components/script/dom/xrsession.rs

```diff
@@ -680,9 +680,7 @@ impl XRSessionMethods for XRSession {
             }
             // https://github.com/immersive-web/layers/issues/189
             if init.layers.is_some() {
```

Comment:
```suggestion
      if self.has_layers_feature() || init.layers.is_some() {
          return Err(Error::NotSupported);
      }
```

Reply:
ahh ! got it !! yes!

Reply:
It's better to make changes locally and apply them from here, because applying them directly often results in incorrect changes like this.